### DISCUSSION TITLE
Navigation directions as notifications for connected smartwatch or smartband

### DISCUSSION THREAD

**Discussion by damianofalcioni** [organicmaps/organicmaps#4324] [Ideas]
Hi All,
I would like to share with you a useful and simple idea that I do not have found applied in any navigation app.
The idea is to create an android notification every time there is a new navigation direction/instruction and immediately remove it from the notifications. In this way the user will normally not see and notice the notification (and they will not go to bloat the smartphone), but in case a smartwatch or smartband is connected to the phone, it will receive and display the notification with the direction to follow.
This feature could be optionally enabled from the settings and will have a big audience as every smartwatch or cheapest band is able to receive notifications.
This will be really useful when you have to follow the directions without looking always at the phone, like while running or cycling.
Thanks,
Damiano

**Comment by biodranik** [comment]
Won't announcing the next turn using voice work better?

**Reply by damianofalcioni** [reply]
in my experience, if you run or cycle without connected earphones is almost impossible to ear the voice instructions, especially if the smartphone is in a pocket

**Reply by biodranik** [reply]
Is using a watch easier/better than fixing the device on the wheel?

In any case, it would be great to make a native watch support, without any hacks.

**Reply by damianofalcioni** [reply]
When cycling, if you have the support on the bike you look the path directly, but also in this case it will be usefull in order to not keep the screen always on, draining  the battery. Also in this case you doesn't look always at the path and a vibration on the  watch is more easy to recognise and read then a voice instruction (especially in city traffic).
Same when you run, where you usually do not want to care about the smartphone and put in a fanny pack or pocket.
Also consider that the use of smartwatch or smart band is common during such activities in order to measure heart rate or steps, and I thought about the notifications because is guaranteed to work on all the devices, also the cheapest ones.

**Reply by GlenCThomas** [reply]
I would like to see this too. My use case is navigating on foot in cities. Regularly looking at the phone screen marks me out as a lost tourist, and the voice instructions are not usable from the pocket with traffic near.

Smart watch notifications buzzing are great for street turns on foot, and I used it a lot with OSMAnd. OrganicMaps is a much better app but I miss the notifications.
